lib/server/routes/users.js

```diff
@@ -360,22 +377,31 @@ UsersRouter.prototype.destroyUser = function (req, res, next) {
 
     user.deactivator = deactivator;
 
-    let profile = self.config.server.public || self.config.server;
-    let host = profile.host;
-    let port = [443, 80].indexOf(profile.port) === -1 ?
-      ':' + profile.port :
-      '';
-    let proto = self.config.server.ssl &&
-      self.config.server.ssl.cert &&
-      self.config.server.ssl.key ?
-      'https:' :
-      'http:';
-
-    self.mailer.dispatchSendGrid(user.email, 'delete', {
-      token: user.deactivator,
-      redirect,
-      url: proto + '//' + host + port
-    }, function (err) {
+    sendGridMail.setApiKey(self.config.mailer.sendgrid.api_key);
+
+    const emailData = {
+      to: user.email,
+      from: {
+        email: 'activate@internxt.com',
+        name: 'Internxt',
+      },
+      subject: '',
+      text: 'deactivate acount',
+      html: 'deactivate acount',
+      personalizations: [
+        {
+          to: [
+            {
+              email: user.email,
+            },
+          ],
+          dynamic_template_data: { email: user.email, deletetion_url: redirect },
+        },
+      ],
+      template_id: self.config.mailer.sendgrid.delete_template_id,
```

Comment:
Same as in drive-server please add a config so that we can enable sandbox mode if it's not prod ✌️

Reply:
Okay! 👍